Web/Types.hs

```diff
@@ -19,6 +19,11 @@ data LandingPageWithRecords = LandingPageWithRecords
 
 --
 
+data CompanyWithRecords = CompanyWithRecords
```

Comment:
`fetchRelated` could work here, but I find that since `fetchRelated` is limited to one level of nesting, it's easier to start already with the `FooWithRecords` pattern